tests/common/dualtor/dual_tor_mock.py

```diff
@@ -357,8 +366,8 @@ def apply_peer_switch_table_to_dut(cleanup_mocked_configs, rand_selected_dut, mo
           'redis-cli -n 4 HDEL"{}" "{}" "{}"'.format(device_meta_key, 'subtype', 'DualToR'),
           'redis-cli -n 4 HDEL "{}" "{}" "{}"'.format(device_meta_key, 'peer_switch', peer_switch_hostname)]
     dut.shell_cmds(cmds=cmds)
-    if dut.get_asic_name() == 'th2':
-        # Restart swss on TH2 platform
+    if dut.get_asic_name() in ['th2', 'td3']:
```

Comment:
These code looks like same, could we form a function to reuse?

Reply:
Fixed. Thanks